aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -136,7 +136,7 @@ static void mean_kernel_impl(TensorIterator& iter) {
 }
 
 static void std_var_kernel_impl(TensorIterator &iter, bool unbiased, bool take_sqrt) {
-  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "std_cpu", [&] {
+  AT_DISPATCH_FLOATING_TYPES_AND2(kBFloat16, kHalf, iter.dtype(), "std_cpu", [&] {
```

Comment:
Tests of std and var exist in test_torch but would need to be updated to take a dtype variant